### PR TITLE
try catch jquery cdn

### PR DESCRIPTION
**Pull request by camilothomas**
I used try/catch on the axios get statement in client.js
I added the jquery cdn because file we were using was causing errors

### CODE REVIEW

File: client/src/client.js [hunk 1]
```diff
@@ -86,7 +86,28 @@ function line_operations(a_line) {
  ***********************************************************/
 async function two_lines() {
   // calling our lambda function
-  var cr_call = await axios.get('https://91duv1eln6.execute-api.us-east-1.amazonaws.com/dev/chartdb');
+  //var cr_call;
```
Comment (brandon-lind) [inline client/src/client.js]: It is best practice to not leave commented code in your check-in. You can always use git history if you need to get something back.

File: client/src/client.js [hunk 2]
```diff
@@ -86,7 +86,28 @@ function line_operations(a_line) {
...
+  try {
+    var cr_call = await axios.get('https://91duv1eln6.execute-api.us-east-1.amazonaws.com/dev/chartdb');
```
Comment (brandon-lind) [inline client/src/client.js]: Best practice is to use let (or in this case const) instead of var.

File: client/src/index.html [hunk 3]
```diff
@@ -167,7 +167,10 @@ <h2>Section title</h2>
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="./Dashboard Template for Bootstrap_files/jquery-3.3.1.slim.min.js.download" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script 
+      src="https://code.jquery.com/jquery-3.3.1.min.js"
```
Comment (brandon-lind) [inline client/src/index.html]: You can knock off the https: protocol and just have it be src="//code.jquery.com/jquery-3.3.1.min.js" to allow this to work with both http and https.

File: client/src/index.html [hunk 4]
```diff
@@ -108,7 +108,7 @@ <h6 class="sidebar-heading d-flex justify-content-between align-items-center px-
 
         <main role="main" class="col-md-9 ml-sm-auto col-lg-10 px-4"><div class="chartjs-size-monitor" style="position: absolute; left: 0px; top: 0px; right: 0px; bottom: 0px; overflow: hidden; pointer-events: none; visibility: hidden; z-index: -1;"><div class="chartjs-size-monitor-expand" style="position:absolute;left:0;top:0;right:0;bottom:0;overflow:hidden;pointer-events:none;visibility:hidden;z-index:-1;"><div style="position:absolute;width:1000000px;height:1000000px;left:0;top:0"></div></div><div class="chartjs-size-monitor-shrink" style="position:absolute;left:0;top:0;right:0;bottom:0;overflow:hidden;pointer-events:none;visibility:hidden;z-index:-1;"><div style="position:absolute;width:200%;height:200%;left:0; top:0"></div></div></div>
           <div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pt-3 pb-2 mb-3 border-bottom">
-            <h1 class="h2">Trello KPI Dashboard</h1>
+            <h1 class="h2"><p id="axios_error">Trello KPI Dashboard</p></h1>
```
Comment (brandon-lind) [inline client/src/index.html]: Normally you wouldn't swap a header. You might want to add an empty div with this id. <div id="axios_error"></div>

File: client/src/client.js (hunk 1 above)
Comment (camilothomas) [reply client/src/client.js]: I got rid of all of the commented code

File: client/src/client.js (hunk 2 above)
Comment (camilothomas) [reply client/src/client.js]: I'm having trouble with this one.  If I change this to const or let, they both go into catch and can't identify the variable later in the function.  I tried to change the way I was doing the axios get request.  Using .then and .catch, but I ran into trouble that way too.

File: client/src/index.html (hunk 3 above)
Comment (camilothomas) [reply client/src/index.html]: Having trouble with this one.  It just isn't working.  I remove http: but it cannot find the file.

File: client/src/index.html (hunk 4 above)
Comment (camilothomas) [reply client/src/index.html]: I think that my new solution for this is better.  It's hard to put things into the canvas on the template that is designed for the chart, so I am just using the chart title.  That way it is in the correct element.
fail

**Pull request merged**